packages/fluentui/react-northstar/src/components/Alert/Alert.tsx

```diff
@@ -96,10 +111,12 @@ export interface AlertProps extends UIComponentProps, ContentComponentProps<Shor
   body?: ShorthandValue<BoxProps>;
 }
 
```

Comment:
Let`s also update the type of `accesibility` props, something like `accessibility?: Accessibility<AlertBehaviorProps>`

Reply:
it's already `accessibility?: Accessibility;`

Should change something else?

Reply:
Yep, it should be something like 
```
accessibility?: Accessibility<AlertBehaviorProps>;
```

https://github.com/microsoft/fluentui/pull/12146/files please check this on for reference

Reply:
Added `AlertBehaviorProps`